tensorflow/lite/tools/optimize/operator_property.cc

```diff
@@ -372,6 +372,11 @@ OperatorProperty GetOperatorProperty(const ModelT* model, int subgraph_index,
       property.outputs = {{0, {}}};
       property.restrict_same_input_output_scale = true;
       property.version = 2;
```

Comment:
Missing a break statement here.

This will fix breakages in internal tests. Creating a bug to increase Transpose test coverage to prevent something like this from happening.

Reply:
Hmm, ithe tests ran locally, succeeding with and without the break statement. I added the break statement now though!

Reply:
Ok. That's what I expected. There does need to be increased test coverage for the transpose op. Thanks for running it again!